src/api/photos/index.ts

```diff
@@ -37,6 +37,16 @@ export const buildRouter = (controller: PhotosController): FastifyRouter => {
         },
         controller.postPhoto.bind(controller),
       );
+      server.post<{ Body: CreatePhotoType }>(
+        '/maybe-existing-photo',
```

Comment:
`/photo/:id/exists`

Let's try to be RESTful

Reply:
We don't know the photoID here, and POST - /exists already exists as endpoint

Reply:
okay, lets try with `/photo/exists` then